tsconfig.json

```diff
@@ -2,7 +2,11 @@
     "compilerOptions": {
         "target": "ES6",
         "outDir": "./built", //removed "module": "CommonJS" to resolve Uncaught ReferenceError: exports is not defined
```

Comment:
It's better to exclude out dirs from git.

Reply:
That makes sense. I guess I need to find a way for Heroku to compile the TS to JS.

Reply:
UPDATE. I added a build script that Heroku picks up, so I no longer need to add the .js to source control.
https://github.com/jsolly/GeoAsteroids/commit/87190ab8fceb3d676ae0503796e2f8376e43b8e9